src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+/**
+ * TO DO:
+ * Update EXPECTED.TXT
+ */
+
+import duke.command.Command;
+import duke.command.ExecutedCommand;
+import duke.tasks.*;
+import duke.parser.Parser;
+import duke.ui.UI;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    /**
+     * Greeting art
+     */
```

Comment:
Since the variable name is `greetingArt`, this comment becomes a bit redundant. Consider removing the comment.